src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/autowiring_classes.php

```diff
@@ -240,7 +244,8 @@ public function setFoo(Foo $foo)
         // should be called
     }
 
-    /** @inheritdoc*/ // <- brackets are missing on purpose
+    /** {@inheritdoc}*/ // <- brackets are missing on purpose
```

Comment:
please read the comment :)

Reply:
Arf bad idea too run cs-fixer on these files :/